TextToSpeech/doc/TextToSpeechPlugin.md

```diff
@@ -240,7 +240,7 @@ TextToSpeech interface events:
 | Event | Description |
 | :-------- | :-------- |
 | [onspeechstart](#event.onspeechstart) | Notifies when speech starts |
-| [onspeechcancel](#event.onspeechcancel) | Notifies when speech is cancelled |
```

Comment:
We might need this cancelling functionality when we support queuing in TTS. I understand like the current requirement is to support only the preemptive speeches.

@ramasat could confirm.

When queuing is supported we should have a call to cancell all speeches. And then it would make sense to send "canceled" events to the speeches from the queue, the progressing speech would get "interrupted" event.

Reply:
Queue'ing is required for "offline" TTS. If we already have the support, I would suggest adding the cancel event for it. If not, we can plan to add both queue'ing and canceled event together in a separate commit later.

Reply:
> Queue'ing is required for "offline" TTS. If we already have the support, I would suggest adding the cancel event for it. If not, we can plan to add both queue'ing and canceled event together in a separate commit later.

We will add the cancel event  as part of queue'ing use case (Offline TTS) . I feel that would be better .

Reply:
Ok, sounds good.